core/const.py

```diff
@@ -870,3 +870,30 @@ class ITLRegion(StrEnum):
         "RiskOwnerRole": "P{i}",
     },
 }
+
+# maps the financial year's start tear, or a month, back to original col letter for Outcomes
+# non-footfall outcomes are mapped by the financial year's start year, but footfall outcomes just by the month
+FINANCIAL_YEAR_OR_MONTH_TO_ORIGINAL_COLUMN_LETTER_FOR_OUTCOMES = {
+    "2020": "F{i}",
+    "2021": "G{i}",
+    "2022": "H{i}",
+    "2023": "I{i}",
+    "2024": "J{i}",
+    "2025": "K{i}",
+    "2026": "L{i}",
+    "2027": "M{i}",
+    "2028": "N{i}",
+    "2029": "O{i}",
+    "April": "D{i}",
+    "May": "E{i}",
+    "June": "F{i}",
+    "July": "G{i}",
+    "August": "H{i}",
+    "September": "I{i}",
+    "October": "J{i}",
+    "November": "K{i}",
+    "December": "L{i}",
+    "January": "M{i}",
+    "February": "N{i}",
+    "March": "O{i}",
+}
```

Comment:
Can we split this out into two dictionaries, keeping the months and years as integers. 

i.e.
`YEAR_TO_OUTCOME_COLUMN` and `MONTH_TO_FOOTFALL_OUTCOME_COLUMN`